initial_setup/__init__.py

```diff
@@ -50,6 +40,24 @@ class InitialSetupError(Exception):
                                 "selinux",
                                 "firewall"]
 
+CONFIGURATION = """
+# Specific configuration for initial setup
+
+[Anaconda]
+kickstart_modules =
+     org.fedoraproject.Anaconda.Modules.Timezone
+     org.fedoraproject.Anaconda.Modules.Network
+     org.fedoraproject.Anaconda.Modules.Localization
+     org.fedoraproject.Anaconda.Modules.Security
+     org.fedoraproject.Anaconda.Modules.Users
+     org.fedoraproject.Anaconda.Modules.Services
+
+"""
+
+# Read and validate the configuration.
+conf.get_parser().read_string(CONFIGURATION)
```

Comment:
So this is getting a parser from some variable in some Anaconda's internal module, using it to parse and validate configuration and then never use it again? That looks really ugly to me. Shouldn't the `conf` be created here and just passed to some other object?

Reply:
The configuration was moved to a file.